docs/framework/deployment/guide-for-administrators.md

```diff
@@ -228,7 +228,7 @@ This step-by-step article describes how a system administrator can deploy the [!
  The following log files are generated during [!INCLUDE[net_v45](../../../includes/net-v45-md.md)] setup:  
   
  %temp%\Microsoft .NET Framework 4.5*.txt  
```

Comment:
shouldn't the same change be done here too?

Reply:
@mairaw, the change was to remove the "\" before the * character, so both lines now correspond. I think that this is OK to merge, since the build looks OK, and the log is placed in the %temp% directory, rather than in a subdirectory of %temp%.

Reply:
good catch @rpetrusha! I was somehow looking in reverse mode.